serial_monitor_to_file.py

```diff
@@ -0,0 +1,55 @@
+#imports
+import serial.tools.list_ports
+from datetime import datetime
+
+#list out all ports in use
+ports: list = serial.tools.list_ports.comports()
+serial_instance = serial.Serial()
+active_port: str = ""
+list_of_recieved_messagdes: str = ""
```

Comment:
Det er vel ikke en liste, som navnet siger? Og ville det i grunden ikke være lidt flottere med en reel liste, som vi bare appender til, i stedet for at gøre det hele i en string med newlines?

Reply:
Derudover synes jeg received messages er tiltrækkeligt 👯